ajax/libs/driver.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "driver.js",
+  "filename": "driver.min.js",
+  "version": "0.5.0",
+  "description": "A light-weight, no-dependency, vanilla JavaScript library to drive the user's focus across the page",
+  "homepage": "http://kamranahmed.info/driver",
```

Comment:
Let's use `https`
What do you think?
Many thanks

Reply:
Nice catch ：+1: